.github/workflows/concrete_python_release_cpu.yml

```diff
@@ -183,17 +182,17 @@ jobs:
           $PYTHON -m venv .venv && . .venv/bin/activate
 
           # Install requirements
-          pip install -r $CONCRETE_PYTHON/requirements.txt
-          pip install -r $CONCRETE_PYTHON/requirements.dev.txt
+          pip install -r "${CONCRETE_PYTHON}"/requirements.txt
+          pip install -r "${CONCRETE_PYTHON}"/requirements.dev.txt
 
           # Build python bindings of concrete compiler
-          cd $CONCRETE_COMPILER
+          cd "${CONCRETE_COMPILER}" || exit
```

Comment:
I don't think `exit` is useful here, if `cd` command fails then it will output a non-zero exit code and the step will automatically fail. Unless I'm missing something here due to double-quoting :thinking:

Reply:
shellcheck not mine

Reply:
actually don't known why I just apply suggestion like a dumb